dftimewolf/lib/collectors/aws.py

```diff
@@ -0,0 +1,252 @@
+          new_volume, self._FindNextAvailableDeviceName())
+      print('Volume {0:s} successfully copied to {1:s}'.format(
+          volume.volume_id, new_volume.volume_id))
+      self.state.output.append((self.analysis_vm.name, new_volume))
+
+  # pylint: disable=arguments-differ,too-many-arguments
+  def SetUp(self,
+            remote_profile_name,
+            remote_zone,
+            incident_id,
+            remote_instance_id=None,
+            volume_ids=None,
+            all_volumes=False,
+            analysis_profile_name=None,
+            analysis_zone=None,
+            boot_volume_size=50,
+            cpu_cores=16,
+            ami='ami-0013b3aa57f8a4331'):
+    """Sets up an Amazon web Services (AWS) collector.
+
+    This method creates and starts an analysis VM in the AWS account and
+    selects volumes to copy from the target instance / list of volumes passed
+    in parameter.
+
+    If volume_ids is specified, it will copy the corresponding volumes from the
+    account, ignoring volumes belonging to any specific instances.
+
+    If remote_instance_id is specified, two behaviors are possible:
+    * If no other parameters are specified, it will select the instance's boot
+      volume.
+    * if all_volumes is set to True, it will select all volumes in the account
+      that are attached to the instance.
+
+    volume_ids takes precedence over remote_instance_id.
+
+    Args:
+      remote_profile_name (str): The AWS account in which the
+          volume(s) exist(s). This is the profile name that is defined in
+          your AWS credentials file.
+      remote_zone (str): The AWS zone in which the source volume(s) exist(s).
+      incident_id (str): Incident identifier used to name the analysis VM.
+      remote_instance_id (str): Optional. Instance ID that needs forensicating.
+      volume_ids (str): Optional. Comma-separated list of volume ids to
+          copy.
+      all_volumes (bool): Optional. True if all volumes attached to the source
+          instance should be copied.
+      analysis_profile_name (str): Optional. The AWS account in which to
+          create the analysis VM. This is the profile name that is defined in
+          your AWS credentials file.
+      analysis_zone (str): Optional. The AWS zone in which to create the VM.
+          If not specified, the VM will be created in the same zone where the
+          volume(s) exist(s).
+      boot_volume_size (int): Optional. The size (in GB) of the boot volume
+          for the analysis VM. Default is 50 GB.
+      cpu_cores (int): Optional. The number of CPU cores to use for the
+          analysis VM. Default is 16.
+      ami (str): Optional. The Amazon Machine Image ID to use to create the
+          analysis VM. Default is Ubuntu 18.04 TLS.
+    """
+
+    if not (remote_profile_name and remote_zone):
+      self.state.AddError('You must specify "remote_profile_name" and "zone" '
+                          'parameters', critical=True)
+
```

Comment:
You need to return right after adding the error (pending https://github.com/log2timeline/dftimewolf/issues/80)